onprc_ehr/src/org/labkey/onprc_ehr/issues/RestrictedIssueProviderImpl.java

```diff
@@ -80,6 +80,10 @@ private String getPropMapName(String issueDefName)
     @Override
     public boolean hasPermission(User user, @NotNull Issue issue, List<Issue> relatedIssues, List<ValidationError> errors)
     {
+        // Site admins always have access
+        if (user.isInSiteAdminGroup())
```

Comment:
I believe that `user.hasSiteAdminPermission()` is the preferred check

Reply:
great, thanks!